internal/store/store.go

```diff
@@ -273,11 +273,31 @@ func (bs *BlockStore) LoadBlockCommit(height int64) *types.Commit {
 	}
 	commit, err := types.CommitFromProto(pbc)
 	if err != nil {
-		panic(fmt.Errorf("error reading block commit: %w", err))
+		panic(fmt.Errorf("error from proto block commit: %w", err))
```

Comment:
```suggestion
		panic(fmt.Errorf("converting commit to proto: %w", err))
```

It seems redundant to include "error" in the text. It's an error (in this case a panic), that much will be clear. 🙂